src/app/src/components/Sidebar.js

```diff
@@ -1,5 +1,172 @@
+            <BasemapLayers />
+        </Flex>
+    );
+}
+
+function TitleBar() {
+    return (
+        <Flex align='center' p={4} mb={2}>
+            <Icon color='white' fontSize='2xl' strokeWidth={1} as={MenuIcon} />
+            <Heading size='md' color='white' ml={6}>
+                Boundary Sync
+            </Heading>
+        </Flex>
+    );
+}
+
+function ReferenceLayers() {
+    return (
+        <Box ml={marginLeft} mt={6} mb={6}>
+            <Flex mb={4} align='center'>
+                <Heading variant='sidebar' size='sm'>
+                    Reference Layers
+                </Heading>
+                <Icon color='white' as={HelpIcon} ml={2} />
+            </Flex>
+            <Button
+                bg='gray.500'
+                color='white'
+                variant='button'
+                leftIcon={<Icon as={PlusIcon} />}
+            >
+                Upload file
+            </Button>
+        </Box>
+    );
+}
+
+function BasemapLayers() {
+    const [satelliteMode, setSatelliteMode] = useState(false);
+    const [visibility, setVisibility] = useState(() =>
+        basemapLayers.reduce(
+            (visibility, layer) => ({
+                ...visibility,
+                [layer]: true,
+            }),
+            {}
+        )
+    );
+
+    const toggleVisibility = layer => {
+        setVisibility(visibility => ({
+            ...visibility,
+            [layer]: !visibility[layer],
+        }));
+    };
+
+    return (
+        <Box ml={marginLeft} mr={marginLeft} mt={6}>
+            <Heading variant='sidebar' size='sm'>
+                Basemap Layers
+            </Heading>
+            <Flex direction='column' align='flex-start' mt={4}>
+                {basemapLayers.map(layer => (
+                    <VisibilityButton
+                        key={layer}
+                        visible={visibility[layer]}
+                        onChange={() => toggleVisibility(layer)}
+                        label={layer}
+                    />
+                ))}
+            </Flex>
+            <Flex mt={2}>
+                <ImageButton
+                    image={BasemapDefaultImage}
+                    label='Default'
+                    selected={!satelliteMode}
+                    onClick={() => setSatelliteMode(false)}
+                />
+                <ImageButton
+                    image={BasemapSatelliteImage}
+                    label='Satellite'
+                    selected={satelliteMode}
+                    onClick={() => setSatelliteMode(true)}
+                />
+            </Flex>
+        </Box>
+    );
+}
+
+function VisibilityButton({ label, visible, onChange }) {
+    return (
+        <Button
+            mb={1}
+            leftIcon={
+                <Flex
+                    color='white'
+                    borderRadius='50%'
+                    bg={visible ? 'gray.500' : 'gray.600'}
+                    align='center'
+                    mr={2}
```

Comment:
This seems like a good spot to use [the Center component](https://chakra-ui.com/docs/components/center)

```suggestion
                <Center
                    color='white'
                    bg={visible ? 'gray.500' : 'gray.600'}
                    mr={2}
```

Reply:
~Center~ → Circle